src/app/tool/seed/faq/faq.component.html

```diff
@@ -117,7 +116,7 @@ <h4 class="mat-h4 mb-4">What is the estimated cost to implement and maintain the
         <li><a href="https://opentech.eco/" target="blank">Open</a></li>
         <li><a href="https://psdconsulting.com/" target="blank">PSD Consulting</a>.</li>
```

Comment:
should we remove PSD here as well since OEP was their connection to SEED?

Reply:
sure!